src/class-elements/private-getter-shadowed-by-getter-on-nested-class.case

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2019 Caio Lima (Igalia SL). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: PrivateName can be shadowed on inner classes by a private getter (private getter)
```

Comment:
nit: `private getter (private getter)`.

Perhaps simplify the general `desc`s?

I know this one is about 2 private getters but this might be a bit distracting. Maybe just use something like: `by two private getters` here and in other cases things like `by a private getter and setter`, `by a private method and a private getter` avoiding the parentheses? 

Avoid this change if it takes too long, it's a nit pick only.

Reply:
I agree. I'll change them